h/views/client.py

```diff
@@ -132,6 +176,20 @@ def absolute_asset_urls(bundle_name):
     }
 
 
+@view_config(route_name='embed',
+             has_feature_flag='use_client_boot_script',
+             http_cache=60 * 5)
```

Comment:
Hmm, although we should probably not do that while it's controlled by a feature flag. Leave this one for now and we'll follow up once we've deployed it.